src/tiled/createbezierobjecttool.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * createbezierobjecttool.h
+ * Copyright 2014, Martin Ziel <martin.ziel@gmail.com>
+ *
+ * This file is part of Tiled.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CREATEBEZIEROBJECTTOOL_H
+#define CREATEBEZIEROBJECTTOOL_H
+
+#include "createobjecttool.h"
+#include "controlpointshandle.h"
+
+namespace Tiled {
+
+namespace Internal {
+
+class CreateBezierObjectTool : public CreateObjectTool
+{
+    Q_OBJECT
+public:
+    CreateBezierObjectTool(QObject *parent);
+    void startNewMapObject(const QPointF &pos, ObjectGroup *objectGroup);
+    void languageChanged() = 0;
+
+protected:
+    void mouseMovedWhileCreatingObject(const QPointF &pos,
+                                       Qt::KeyboardModifiers modifiers, bool snapToGrid, bool snapToFineGrid);
+    void mousePressedWhileCreatingObject(QGraphicsSceneMouseEvent *event, bool snapToGrid, bool snapToFineGrid);
+    void mouseReleasedWhileCreatingObject(QGraphicsSceneMouseEvent *event, bool snapToGrid, bool snapToFineGrid);
```

Comment:
Even though there are no merge conflicts, I broke this tool because I changed these function signatures in commit cef41290b159ec2fff8c31db4e86e929aef95299. Sorry about that.